README.md

```diff
@@ -1 +1,11 @@
-``` shell
g++ -static lib_sample_use.cpp -std=c++11 -lfastFM -lglog -pthread -I. -L.
```
\ No newline at end of file
+Build:
+
+    > $GCC_MUSL_ROOT/x86_64-linux-musl-g++ -static -std=c++11 lib_sample_use.cpp libfastFM.a libglog.a -o foo
```

Comment:
:-( Thanks, I'll ping you as soon as I have made up my mind about the next steps.